### DISCUSSION TITLE
github enterprise server event

### DISCUSSION THREAD

**Discussion by kimkijeong1** [community/community#53206] [Enterprise]
### Select Topic Area

Question

### Body

When I use git commands on my local machine, github doesn't catch events. When I enter the github site, it catches the event well.
When using the regular free version of Github, it works fine locally, but when using the enterprise server version, events are not caught.
Is there a setting in the settings that can receive events?

**Comment by samus-aran** [comment]
Hi @kimkijeong1 

I think we need a little more information for others to help troubleshoot. Do you have any errors? Have you check the [Enterprise Server documentation troubleshooting Actions](https://docs.github.com/en/enterprise-server@3.8/admin/github-actions/advanced-configuration-and-troubleshooting/troubleshooting-github-actions-for-your-enterprise)? I'll include the [About actions in your enterprise](https://docs.github.com/en/enterprise-server@3.8/admin/github-actions/managing-access-to-actions-from-githubcom/about-using-actions-in-your-enterprise) documentation if this would help.
